### PR TITLE
String drop 2023-01-03

### PR DESCRIPTION
**Pull request by dan-alcantara-neeva**
Update strings from the last week of December.

### CODE REVIEW

File: app/src/main/res/values-de-rDE/strings.xml [hunk 1]
```diff
@@ -988,7 +989,7 @@
     <string name="welcomeflow_annual_premium_plan">Premium\njährlich</string>
 
     <!-- [WelcomeFlow] Name of the Monthly Premium Subscription Plan. -->
-    <string name="welcomeflow_monthly_premium_plan">Monatlich Jährlich</string>
+    <string name="welcomeflow_monthly_premium_plan">Premium\nmonatlich</string>
```
Comment (andjd) [inline app/src/main/res/values-de-rDE/strings.xml:992]: do we want hardcoded newlines in here?

File: app/src/main/res/values-en-rGB/strings.xml [hunk 2]
```diff
@@ -988,7 +989,7 @@
     <string name="welcomeflow_annual_premium_plan">Premium\nAnnual</string>
 
     <!-- [WelcomeFlow] Name of the Monthly Premium Subscription Plan. -->
-    <string name="welcomeflow_monthly_premium_plan">Monthly Annual</string>
+    <string name="welcomeflow_monthly_premium_plan">Premium\nMonthly</string>
```
Comment (andjd) [inline app/src/main/res/values-en-rGB/strings.xml:992]: Same

**Review by andjd — APPROVED** [review]

File: app/src/main/res/values-de-rDE/strings.xml (hunk 1 above)
Comment (dan-alcantara-neeva) [reply app/src/main/res/values-de-rDE/strings.xml]: Removed from Smartling's memory, as per conversation on Slack:
https://neevaco.slack.com/archives/C02RYG8LY8H/p1672770627823909

tl;dr: iOS hard-coded the newlines in and Smartling gives them back to us even though we don't add a `\n` ourselves.  Removed the newline in our translations, pending input from Hwi.

File: app/src/main/res/values-en-rGB/strings.xml (hunk 2 above)
Comment (dan-alcantara-neeva) [reply app/src/main/res/values-en-rGB/strings.xml]: Same as above
